db/migrate/20150330203929_create_reviews.rb

```diff
@@ -0,0 +1,10 @@
+class CreateReviews < ActiveRecord::Migration
+  def change
+    create_table :reviews do |t|
+      t.integer :user_id, null: false
+      t.text :body, null: false
+      t.integer :game_rating
+      t.integer :upvotes
```

Comment:
we took them out with a migration. After we thought about it and it was to complicated